test/integration/StrykerJestRunnerSpec.ts

```diff
@@ -0,0 +1,55 @@
+import JestConfigEditor from '../../src/JestConfigEditor';
+import { Config } from 'stryker-api/config';
+import { RunnerOptions, RunStatus, TestStatus } from 'stryker-api/test_runner';
+import * as sinon from 'sinon';
+import { expect } from 'chai';
+import JestTestRunner from '../../src/JestTestRunner';
+import * as path from 'path';
+
+describe('StrykerJestRunner', () => {
+  const jestTestRunnerRoot = process.cwd();
+
+  let jestConfigEditor: JestConfigEditor;
+  let runOptions: RunnerOptions;
+  let projectRoot = path.join(jestTestRunnerRoot, 'testResources', 'reactProject');
+  let getProjectRootStub: sinon.SinonStub;
+
+  let sandbox: sinon.SinonSandbox;
+
+  process.env.BABEL_ENV = 'test';
+
+  beforeEach(() => {
```

Comment:
Having a beforeEach and afterEach for just one spec seems like overkill.

Reply:
But it makes it easier to add tests later on right? Why not leave it? Also: this file already has  more tests.

Reply:
It had only one test before, I added a test in the last revision.